src/schemas/complaint-tool/schema.js

```diff
@@ -174,34 +179,40 @@ let schema = {
         type: 'object',
         properties: {
           required: ['name'], // address or facilityCode are also required on FE
-          address: { // TRANSLATE: given 'address': '123 Main St.' and 'address2': 'APT 23' then the final translated 'address' should be '123 Main St., APT 23'
-            type: 'string',
-            minLength: 1,
-            maxLength: 126 // address + address2 length must be < 255
-          },
-          address2: {
-            type: 'string',
-            minLength: 1,
-            maxLength: 126 // address + address2 length must be < 255
-          },
-          city: {
-            type: 'string',
-            minLength: 1,
-            maxLength: 255
-          },
-          state: { // backend requires the full state names for the school address
-            type: 'string',
-            'enum': allStates.map(state => state.full),
-          },
-          postalCode: {  // TYPE: text (255)
-            type: 'string',
-            pattern: '^\\d{5}$' // common definition pattern (meets submission requirements)
-          },
-          country: {
-            type: 'string',
-            'enum': ['US'], // Only 'US' addresses are supported
-            enumNames: ['United States'],
-            default: 'US'
+          address: {
+            type: 'object',
+            required: ['street', 'city', 'state', 'country', 'postalCode'],
+            properties: {
+              street: {
+                type: 'string',
+                minLength: 1,
+                maxLength: 126 // address + address2 length must be < 255
+              },
+              street2: {
+                type: 'string',
+                minLength: 1,
+                maxLength: 126 // address + address2 length must be < 255
+              },
+              city: {
+                type: 'string',
+                minLength: 1,
+                maxLength: 255
+              },
+              state: { // backend requires the full state names for the school address
+                type: 'string',
+                'enum': allStates.map(state => state.full),
```

Comment:
in the sample request (which was accepted by the salesforce server) the state abbreviation for the school address is used, doesn't look like full state names are required

Reply:
Thanks @lihanli, updated.
Stakeholders indicated: 
> The state dropdown for the individual completing this form is abbreviated while the state dropdown for the school is spelled out.

So I've left a TODO comment about confirming this with them.